File: Subastas-app/subastas-client/src/dto/auth/RegisterRequestDTO.ts
export interface RegisterRequestDTO {
  nombre: string;
  apellido: string;
  mail: string;
  password: string;

  frenteDNIUrl: string;
  dorsoDNIUrl: string;

  domicilio: {
    provincia: string;
    ciudad: string;
    cp: string;
    direccion: string;
  };
}

File: Subastas-app/subastas-api/src/main/java/com/subastas/subastas_api/DTO/auth/RegisterRequestDTO.java
package com.subastas.subastas_api.DTO.auth;

import com.subastas.subastas_api.model.Domicilio;

public class RegisterRequestDTO {

    private String nombre;
    private String apellido;
    private String mail;
    private String password;

    private String frenteDNIUrl;
    private String dorsoDNIUrl;

    private Domicilio domicilio;

    public RegisterRequestDTO() {
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public String getMail() {
        return mail;
    }

    public String getPassword() {
        return password;
    }

    public String getFrenteDNIUrl() {
        return frenteDNIUrl;
    }

    public String getDorsoDNIUrl() {
        return dorsoDNIUrl;
    }

    public Domicilio getDomicilio() {
        return domicilio;
    }
}